install-manuals/all-services/README.md

```diff
@@ -1,21 +1,20 @@
 # How to run all services
 
-All services are in a `docker compose` file.
+All services are available as docker containers. The preferred way to host is to use a `docker compose` file.
 
-## Prerequisite
+We provide a template in our [run-config](https://github.com/Gamify-IT/run-config) repository. There is also a [manual](https://github.com/Gamify-IT/run-config/hosting-template#setup) how to setup the hosting environment.
```

Comment:
Could you perhaps do it the other way round and pack the setup into the docs and link here from the run-config?

Reply:
I don't know if this is better. Now the manual is at the same place as the files for which it is for.